core/src/main/java/edu/wpi/grip/core/Operation.java

```diff
@@ -23,6 +31,13 @@
     String getDescription();
 
     /**
+     * @return What category the operation falls under.  This is used to organize them in the GUI
+     */
+    default Category getCategory() {
+        return Category.MISCELLANEOUS;
+    }
```

Comment:
Probably not.  "Other" kind of sounds like there are other categories besides the one listed, which isn't true.